Added doctor list to the left hand navigation pane

Amaarillo opened the pull request:
Now all doctors are visiable in the left hand navigation pane together
with nurses and patients

Amaarillo committed:
Added doctor list to the left hand navigation pane

Now all doctors are visiable in the left hand navigation pane together
with nurses and patients

delta-one commented:
Maybe you can add the online-status of the doctors as well like it is with the nurses. It should only be like 2 more lines of code in the pug-template, since the list is kept centrally and includes all users.

Amaarillo committed:
Added online-status for doctors

Hardcoded the online-status for doctors in the same manner as we did for
nurses

Amaarillo committed:
Restructured the left han navigation pane

delta-one commented:
Looks fine, merge it.
